### PR TITLE
Support automatic decoding of Windows-1252

### PR DESCRIPTION
**Pull request by evert**

### CODE REVIEW

File: lib/Parser/MimeDir.php [hunk 1]
```diff
@@ -460,6 +469,9 @@ protected function readProperty($line) {
                 case 'ISO-8859-1' :
```
Comment (staabm) [inline lib/Parser/MimeDir.php]: Side note: what about iso-8859-15?

File: lib/Parser/MimeDir.php [hunk 2]
```diff
@@ -460,6 +469,9 @@ protected function readProperty($line) {
...
                     $property['value'] = utf8_encode($property['value']);
                     break;
+                case 'Windows-1252' :
```
Comment (staabm) [inline lib/Parser/MimeDir.php]: Some people call it cp1252, not sure you want to also handle this "alias"

Comment (evert) [inline lib/Parser/MimeDir.php]: I basically just want to add the character sets on a case-by-case basis. ISO-8859-15 makes a lot of sense, but if there's no software out there that specifically marks files at ISO-8859-15, then I don't really see a strong need to add it.

Basically I just want to wait until people request these specific features.

Comment (staabm) [inline lib/Parser/MimeDir.php]: Okidoki

**Pull request merged**